src/Init/LaunchpadSetup.lua

```diff
@@ -118,14 +118,14 @@ function LaunchpadSetup:activate()
     self.play_record_button:activate()
 end
 
-function LaunchpadSetup:connect_launchpad(pad_name,rotation)
+function LaunchpadSetup:connect_launchpad(pad_in_name,pad_out_name,rotation)
```

Comment:
what do you need the out_device here? why should the render device and the input device differ?